Gemfile

```diff
@@ -75,7 +75,7 @@ gem 'arc-client', git: 'https://github.com/sapcc/arc-client.git'
 gem 'sdoc', '~> 1.1.0', group: :doc
 
 # if you update puma check Dockerfile for Version dependencies!
-gem 'puma', '= 4.3.9', require: false
+gem 'puma', '= 3.12.6', require: false
```

Comment:
Did you notice that you downgrade puma? 😅
Please just add your required libraries without downgrade or upgrade the rest of the libraries.